packages/react-ui-core/src/IncrementDecrementFilter/IncrementDecrementFilter.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component, PropTypes } from 'react'
```

Comment:
Please import PropTypes directly from the lib itself. It will eventually be deprecated from the react package.

```
import PropTypes from `prop-types`
```